chai.js

```diff
@@ -129,7 +129,7 @@ module.exports = function (_chai, util) {
    */
```

Comment:
This file should be excluded from the diff. See the notes around pull requests here: https://github.com/chaijs/chai/blob/master/CONTRIBUTING.md#pull-requests

Namely:

> ```bash
npm test
# when finished running tests...
git checkout chai.js
```